.github/workflows/validate-changelog-content-infra-tests.yml

```diff
@@ -0,0 +1,50 @@
+name: Run content infrastracture tests
+on: pull_request
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Setup Python
+        uses: actions/setup-python@v3
+        with:
+          python-version: '3.10'
+
+      - name:  Get commit hash
+        id: get-commit
+        run: echo commit=$(git rev-parse HEAD) >> $GITHUB_OUTPUT
+
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          repository: demisto/content
+          path: content
+
+
+      - name: Setup Poetry
+        uses: Gr1N/setup-poetry@v8
+
+      - name: Install demisto-SDK from branch
+        run: |
+          cd content
+          poetry install --with ci
+          echo "uninstalling content-master version of demisto-sdk"
+          pip uninstall -y demisto-sdk
```

Comment:
See [this](https://github.com/demisto/demisto-sdk/actions/runs/5641660966/job/15280070757?pr=3369#step:7:214) warning, I'm not sure this is running in the poetry venv. Let's try `poetry run pip uninstall demisto-sdk`, or `poetry remove demisto-sdk` (could be slower, but is more likely to succeed)

Reply:
I used poetry remove demisto-sdk